src/projects.h

```diff
@@ -590,7 +595,7 @@ extern struct PJ_PRIME_MERIDIANS pj_prime_meridians[];
 
 
 #ifdef PJ_LIB__
-#define PROJ_HEAD(id, name) static const char des_##id [] = name
+#define PROJ_HEAD(name, desc) static const char des_##name [] = desc
```

Comment:
Seems to be outside the scope of this PR. I'd prefer you do this separately so it is easier in the future to follow what's been going on.

Reply:
I actually cant remember my own exact justification for this, but it had to do with clarification of some terms somewhere in one of the `PJ_` files.

The problem was rooted in the fact that that the `id` parameter in the old definition of PROJ_HEAD is the same as the `name`  parameter of OPERATION, while the `name` parameter of the old PROJ_HEAD definition had a different meaning.

So probably more in scope than it looks, although I have forgotten the exact reason.